main/lsp/LSPIndexer.h

```diff
@@ -36,18 +36,20 @@ class LSPIndexer final {
      * that we correctly track the latency of canceled & rescheduled typechecking operations. */
     std::vector<std::unique_ptr<Timer>> pendingTypecheckDiagnosticLatencyTimers;
     /** Contains files evicted by `pendingTypecheckUpdates`. Used to make fast path decisions in the immediate past. */
-    UnorderedMap<int, std::shared_ptr<core::File>> evictedFiles;
+    UnorderedMap<core::FileRef, std::shared_ptr<core::File>> evictedFiles;
```

Comment:
It's unlikely we'll see 2^31 files anytime soon, but I don't want to take any chances.